theano/tensor/nnet/tests/test_conv3d2d.py

```diff
@@ -190,3 +217,87 @@ def test_conv3d(border_mode, mode=mode_without_gpu, shared=theano.tensor._shared
     filters = numpy.random.rand(Nf, Tf, C, Hf, Wf).astype('float32')
     utt.verify_grad(lambda s, f: conv3d(s, f, border_mode=border_mode),
                     [signals, filters], eps=1e-1, mode=mode)
+
+
+def get_conv3d_test_cases(experiment='fwd'):
```

Comment:
Why did you added those 2 tests? What wasn't tested in the test above? I don't want to duplicate tests as they are already slow enough.

Otherwise, I'm ok with this PR, but it need a rebase.

thanks.